Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -9,17 +11,60 @@ public static class ArrayOperations
         /// <param name="array">Input array in a random order.</param>
         public static void Sort(int[] array)
         {
-            // ToDo: implement.
+            int temp;
+
+            for (int i = 0; i < array.Length - 1; i++)
+            {
+
+                for (int j = i + 1; j < array.Length; j++)
+                {
+                    if (array [i] > array[j])
+                    {
+
+                        temp = array[i];
+                        array[i] = array[j];
+                        array[j] = temp;
+
+                    }
+
+                }
+
+            }
+            foreach (int sort in array)
+            Console.Write("{0} ", sort);
         }
 
         /// <summary>
         /// Reverse the array elements, first being last and so on.
         /// If array empty or null- don't do anything.
         /// </summary>
         /// <param name="array">Input array in a random order.</param>
-        public static void Reverse(int[] array)
+        public static int[] Reverse(int[] array)
         {
-            // ToDo: implement.
+            int temp;
+
+            // traverse 0 to array length 
+            for (int i = 0; i < array.Length - 1; i++)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'for' block.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4931876)